Gopkg.toml

```diff
@@ -97,3 +102,8 @@ version="v1.4.7"
   name = "github.com/docker/docker"
   revision = "a9fbbdc8dd8794b20af358382ab780559bca589d"
 
+
+[[override]]
+  name = "github.com/jteeuwen/go-bindata"
+  version = "3.0.7"
```

Comment:
I would use `revision = "a0ff2567cfb70903282db057e799fd826784d41d"` which is what we use elsewhere in openshift. You get a nice MustAsset function that helps avoid a lot of error checking.